app/views/content/blog/overcoming-challenges-to-become-a-teacher-hearing-impairment.md

```diff
@@ -38,4 +38,6 @@ The application process went smoothly thanks to the help of my [teacher training
 
 If someone came up to me and asked if they should teach, my answer would be yes and no! No, because teaching is not for everyone and it is a hugely demanding job so it is extremely important to understand the amount of patience, dedication and commitment you will need! However, teaching is a natural skill that anyone can learn as long as you are passionate about the subject and age group you’re teaching. Yes, because teaching allows students to absorb your knowledge and you can watch them flourish throughout the years, which makes the job so fascinating and unique!
 
+Find out about the [support you can get while training to teach if you're disabled](/funding-and-support/if-youre-disabled).
```

Comment:
As with the other blog, I think this is probably better in the indented footnote (ie distinct from the first person story)

Reply:
Agree, line moved to footnote.